cmake/finders/FindFFnvcodec.cmake

```diff
@@ -0,0 +1,96 @@
+#[=======================================================================[.rst
+FindFFnvcodec
+----------
```

Comment:
Make this underline the same length as the text above.

Reply:
Done.